x/programs/rust/wasmlanche-sdk/src/state.rs

```diff
@@ -147,9 +143,8 @@ impl<'a, K: Key> State<'a, K> {
 
         let args_bytes = borsh::to_vec(&args).map_err(|_| StateError::Serialization)?;
 
-        let ptr = unsafe { delete(args_bytes.as_ptr(), args_bytes.len()) };
+        let bytes = unsafe { delete(args_bytes.as_ptr(), args_bytes.len()) };
```

Comment:
nit: any strong opinion on calling this `bytes` ?

Reply:
I guess I could call it `bytes_ref` since that's how you're supposed to treat it, but that kind of goes against the convention.